ddtrace/internal/asm.py

```diff
@@ -0,0 +1,263 @@
+# Use @ for labels
```

Comment:
This is really cool! 

One small nit: Can we rename this module. It can be confused with the ASM product 😅

Reply:
Ah good point!